operator/controllers/suite_test.go

```diff
@@ -1,81 +0,0 @@
-/*
```

Comment:
why is this being deleted completely? I think this is needed to instantiate the tests

Reply:
You're right let me add it back again

Reply:
Added, but commented since it contains an error on Zap

Reply:
Great, thank you. I'll fix it up in #43 after this is merged.